cscs-checks/microbenchmarks/alloc_speed/alloc_speed.py

```diff
@@ -16,9 +16,16 @@ def __init__(self, hugepages):
         if hugepages == 'no':
             self.valid_systems += ['kesch:cn', 'kesch:pn']
         else:
-            if self.current_system.name in {'dom', 'daint'}:
+            if self.current_system.name in {'dom', 'daint', 'tiger'}:
                 self.modules = ['craype-hugepages%s' % hugepages]
 
+        if self.current_system.name in {'tiger'}:
+            self.extra_resources = {
+                'mem-per-cpu': {
+                    'mem_per_cpu': 0
+                }
+            }
```

Comment:
@jgphpc I just noticed that I have read your question regarding this on Slack in the opposite way. You may also set launcher options directly if you wish. I've no strong preference.

Reply:
actually, `extra_resources` is the best way